runtime/gc_glue_java/ConcurrentMarkingDelegate.hpp

```diff
@@ -324,13 +324,21 @@ class MM_ConcurrentMarkingDelegate
 	{
 #if defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING)
 		if (0 != bytesTraced) {
-			_scanClassesMode.setScanClassesMode(MM_ScanClassesMode::SCAN_CLASSES_NEED_TO_BE_EXECUTED); /* need more iterations */
+			setConcurrentScanning(env);/* need more iterations */
 		} else {
 			_scanClassesMode.setScanClassesMode(MM_ScanClassesMode::SCAN_CLASSES_COMPLETE); /* complete for now */
 		}
 #endif /* J9VM_GC_DYNAMIC_CLASS_UNLOADING */
 	}
 
+	MMINLINE void
+	setConcurrentScanning(MM_EnvironmentBase *env)
```

Comment:
I don't have full understanding of these modes, but I don't feel this mode has strict relationship with Concurrent scanning

Reply:
It's specifically for concurrent class scanning, I was following the existing naming convention for Concurrent Class Scanning, for example see existing `startConcurrentScanning` method which relates to Concurrent classes scanning as well.

Reply:
There are also: `isConcurrentScanningComplete` `reportConcurrentScanningMode``concurrentInitializationComplete`

Reply:
These all deal with strictly with concurrent  (class) scanning mode

Reply:
Should I rename these methods?

Reply:
looks ok, afterall